examples/trainercontroller_configs/logging_controller_with_rank.yaml

```
@@ -0,0 +1,33 @@
+controller_metrics:
+  - name: trainer_state
+    class: TrainingState
+  - name: training_loss
+    class: Loss
+  - name: per_process_state
+    class: PerProcessState
+operations:
+  - name: logger_on_log
+    class: LogControl
+    arguments:
+      log_format: 'Epoch: {trainer_state[epoch]:.0f}, Step: {trainer_state[global_step]}, Rank: {per_process_state[rank]}, loss = {training_loss}'
+      log_level: INFO
+  - name: logger_on_save
+    class: LogControl
+    arguments:
+      log_format: |
+        Saving model in huggingface format at samples_seen: {trainer_state[global_step]}
```

```suggestion
        Saving model in huggingface format at samples_seen: {trainer_state[global_step]}
```

It should not be samples_seen, its on a step, isn't it?

---

Addressed.